### DISCUSSION TITLE
How do we measure avg and max transaction latency for hammerdb in sql server?

### DISCUSSION THREAD

**Discussion by bbcc1** [TPC-Council/HammerDB#348] [Q&A] [answered]
I am reading this article https://aws.amazon.com/blogs/modernizing-with-aws/performance-benchmark-sql-server-workload-on-aws-and-azure/ and it describe transaction latency. I am wondering how to get such metrics in hammerdb?

**Comment by sm-shaw** [comment, marked as answer]
Time profiling as described in the documentation [here ](https://www.hammerdb.com/docs/ch04s06.html#d0e1780) and blog [here](https://www.hammerdb.com/blog/uncategorized/hammerdb-v4-1-extended-time-profiling/)
